src/pages/promises/[...slug].js

```diff
@@ -168,8 +168,16 @@ export async function getStaticProps({ params: { slug: slugParam }, locale }) {
   const actNowPage = await wpApi.pages({ slug: "act-now", locale }).first;
 
   const backend = backendFn();
+
   const promisePost = await backend.promises({ id }).first;
+  let factChecks = await backend.factChecks().all;
+  factChecks = factChecks.filter((factCheck) =>
+    promisePost.categories.some(
+      (category) => factCheck.category === category.name
```

Comment:
FactCheck should also be allowed to have multiple categories

Reply:
let me see how that will play out in the google sheet, I wanted to reduce the Ids, but if I am moving it to another sheet, I might have no choice but to introduce ids for fact checks as well.